vlib/v/help/common/vet.txt

```diff
@@ -12,7 +12,8 @@ Options:
 
   -v, -verbose        Enable verbose logging.
 
-  -F                  Report empty and long function declaration (>300 lines).
+  -F                  Report potential function to be inlined, empty, long function
+                      declaration (>300 lines).
```

Comment:
Why not 2 separate option flags, instead of reusing the same option for both?

Reply:
Done.